src/main/java/eu/dissco/backend/database/jooq/tables/NewUser.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * This file is generated by jOOQ.
+ */
+package eu.dissco.backend.database.jooq.tables;
+
+
+import eu.dissco.backend.database.jooq.Keys;
+import eu.dissco.backend.database.jooq.Public;
+import eu.dissco.backend.database.jooq.tables.records.NewUserRecord;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+
+import org.jooq.Field;
+import org.jooq.ForeignKey;
+import org.jooq.Name;
+import org.jooq.Record;
+import org.jooq.Row8;
+import org.jooq.Schema;
+import org.jooq.Table;
+import org.jooq.TableField;
+import org.jooq.TableOptions;
+import org.jooq.UniqueKey;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.jooq.impl.TableImpl;
+
+
+/**
+ * This class is generated by jOOQ.
+ */
+@SuppressWarnings({ "all", "unchecked", "rawtypes" })
+public class NewUser extends TableImpl<NewUserRecord> {
```

Comment:
This class extends NewUserRecord, but NewUserRecord imports NewUser. Is that not circular?
I'm not clear on the relationship between these two classes

Reply:
This code is generated by JOOQ